### ISSUE TITLE
ACL Anthology by Google Scholar

### ISSUE THREAD

**Issue by kenclr** [acl-org/acl-anthology#126]
This was discussed on Aug 9, 2017. But, this problem is still present. A colleague (with co-authors) had a paper in ACL 2018, after previously writing it in arXiv. So, I was expecting to see a Google Scholar citation alert from the ACL 18 proceedings. Mostly, I was hoping to see other ACL proceedings (CL, TACL, ACL, workshops) that might have cited other of my papers. Mostly, I have viewed such references as important for identifying others who have been working on similar work. The earlier discussion (in 2017) has not been solved. It would be helpful.

**Comment by CTNLP** [comment]
It is possible that Google scholar has not yet re-crawled the Anthology since the publication of the 2018 ACL proceedings. When we last talked to them (the Google Scholar people) in 2017 they told us that the problem was fixed both on our and their side and that they would crawl the anthology every few months. And papers from last years NAACL and TACL are indexed in their aclweb versions (e.g., https://scholar.google.com/scholar?cluster=13945806059840433863&as_sdt=0,5 and https://scholar.google.com/scholar?cluster=8396726957200250116&as_sdt=0,5). So I would still have hope that this is just Google being very sporadic with their updates. Also: there is currently some effort to re-work the Anthology website, which means it might be a waste of time to get Google Scholar to play nice with the current version.
On the other hand, it is now around half a year since the proceedings were published and the indexing should have. So I would suggest the following: I will put this issue on hold until April 02. (too months before NAACL - that way we have a chance to fix this before the next big conference.). If I do not is see the ACL 2018 proceedings in Google Scholar at that point I will talk to our contact person there and try to figure out what is going wrong.
I hope that is an acceptable approach.

**Comment by mjpost** [comment]
Hi—Thanks for filing this. We expect issues with Google Scholar indexing to be resolved when the static rewrite is complete, which is targeted for next month.

Note that I am pretty sure indexing has happened. If you click on the tiny "All X versions" in the Google Scholar search results, the ACL version is there (for the few papers I just spot-checked). Of course, we would like them to be the main result.

**Comment by mjpost** [comment]
Please see [this page](https://github.com/acl-org/acl-anthology/wiki/Google-Scholar-Indexing). We agree this is a serious problem, have plans to fix it, and hope they will be completed very soon.

**Comment by CTNLP** [comment]
The ACL anthology versions of the ACL papers now appear on the google scholar pages. It seems as if it just takes a while for google to crawl the site. Unless there is a good reason not to, I would suggest closing this issue and creating a new one that is specifically for pointing the Google people to the new version of the site, once that is complete and accepted.

**Comment by mjpost** [comment]
Agreed. @kenclr’s paper appears in the Scholar search results. It’s not the first one but that’s beyond our control and in this situation probably shouldn’t be. His second search term still turns up the wrong papers but I think that’s because “acl” isn’t a valid word for the paper.
